jplag/src/main/assembly/jar-with-deps-merge-services.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
+  <!-- copied from jar-with-dependencies (http://maven.apache.org/plugins/maven-assembly-plugin/descriptor-refs.html#jar-with-dependencies) -->
+  <id>jar-with-deps-merge-services</id>
```

Comment:
Jar name should still be `jar-with-dependencies` so adapt accordingly if required.

Reply:
Since there is already a descriptor `jar-with-dependencies` I can't reuse the name, hence the changed name.

Initially I wanted to avoid adding a new dependency into the build, just to do that, but I'll look into shade to come up with something else.